lib/services/platform-service.ts

```diff
@@ -325,7 +329,11 @@ export class PlatformService extends EventEmitter implements IPlatformService {
 			excludedDirs.push(constants.TNS_MODULES_FOLDER_NAME);
 		}
 
-		this.$projectFilesManager.processPlatformSpecificFiles(directoryPath, platform, excludedDirs);
+		const projectFilesConfig: IProjectFilesConfig = {
```

Comment:
this code is already available in this method - some lines above in the first `if`, can you declare it at a single place and use it in both methods where it is required.